coalib/bears/requirements/PackageRequirement.py

```diff
@@ -0,0 +1,43 @@
+from coala_decorators.decorators import generate_eq
+from coala_decorators.decorators import generate_repr
```

Comment:
you can wrap it using braces:
```
from ... import (generate_eq,
        generate_repr)
# or
from ... import (
    generate_eq, generate_repr)
```
coala shouldn't corrected it automatically then^^